package.json

```diff
@@ -61,5 +61,8 @@
 		"pretest:php": "wp-env run cli --env-cwd=/var/www/html/wp-content/themes/$(basename $(pwd)) composer install --no-interaction --no-scripts",
 		"test:php": "wp-env run cli --env-cwd=/var/www/html/wp-content/themes/$(basename $(pwd)) composer exec 'phpunit --do-not-cache-result --verbose'",
 		"wp-env": "wp-env"
+	},
+	"dependencies": {
+		"@formatjs/icu-messageformat-parser": "2.7.8"
```

Comment:
Should this be part of `devDependencies`?

Reply:
Why this additional dependency is required?

Reply:
@thelovekesh the `intl-messageformat-parser` which was being used has been deprecated now and this `@formatjs/icu-messageformat-parser` was suggested as the new updated version of it. Hence the update.

![image](https://github.com/user-attachments/assets/f86ec60c-4c67-469a-a892-7eca57948617)

Reply:
@mchirag2002 Such npm warnings are common. I don't see a reason why we need to include another dependency to just resolve this warning even we don't use it in our project directly.

Reply:
@mchirag2002 

@thelovekesh is right, if it's another package's dependency, that package should update it. I didn't realize it's a dependency not being used directly by us.

Reply:
Even with this change, I am getting these warnings:

![image](https://github.com/user-attachments/assets/cbe42246-890c-43ea-8f62-4056201fd486)

and this package is being used by

```
➜  theme-elementary git:(feature/update-node-modules) npm ls intl-messageformat-parser          
elementary-theme@1.0.0 /home/thelovekesh/Desktop/work/rtCamp/theme-elementary
└─┬ @wordpress/scripts@28.3.0
  └─┬ @wordpress/e2e-test-utils-playwright@1.3.0
    └─┬ lighthouse@10.4.0
      └─┬ intl-messageformat@4.4.0
        └── intl-messageformat-parser@1.8.1
```

Reply:
@thelovekesh Implemented the requested changes in - https://github.com/rtCamp/theme-elementary/pull/479/commits/e068701df05026a1e3efe6a452219aac3192d887